data/every_dream.py

```diff
@@ -135,18 +137,18 @@ def __get_image_for_trainer(self, image_train_item: ImageTrainItem, debug_level=
         if image_train_tmp.cond_dropout is not None:
             example["cond_dropout"] = image_train_tmp.cond_dropout
         example["runt_size"] = image_train_tmp.runt_size
-       
+
         return example
 
     def __update_image_train_items(self, dropout_fraction: float):
         self.image_train_items = self.data_loader.get_shuffled_image_buckets(dropout_fraction)
-        
+
 def build_torch_dataloader(dataset, batch_size) -> torch.utils.data.DataLoader:
     dataloader = torch.utils.data.DataLoader(
         dataset,
-        batch_size=batch_size,
+        batch_size= batch_size,
         shuffle=False,
-        num_workers=4,
+        num_workers=min(batch_size, os.cpu_count()),
```

Comment:
good change, I had been doing something like cpucount/2 before but some point it got set to 4 or I forgot to commit =\